src/main/scala/com/cloudera/sparkts/TimeSeriesRDD.scala

```diff
@@ -118,6 +118,30 @@ class TimeSeriesRDD(val index: DateTimeIndex, parent: RDD[(String, Vector[Double
   }
 
   /**
+   * Takes the first N series in the TimeSeriesRDD and returns a new, reduced, TimeSeriesRDD.
+   * Works by creating an array of series to include and then checks for membership.
+   * @param n the number of series to take (first n taken) n >= 0
+   * @return reduced TimeSeriesRDD
+   */
+  def takeNSeries(n: Int): TimeSeriesRDD = {
```

Comment:
I think I'd keep it in the portfolio optim example then.  It does seem useful, but I'd rather err on the side omitting APIs that we're not 100% sure about, because once they're there, it's hard to take them back.